arrow/compute/scalar_compare.go

```diff
@@ -134,4 +136,50 @@ func RegisterScalarComparisons(reg FunctionRegistry) {
 	reg.AddFunction(ltFn, false)
 	lteFn := makeFlippedCompare("less_equal", gteFn, EmptyFuncDoc)
 	reg.AddFunction(lteFn, false)
+
+	isOrNotNullKns := kernels.IsNullKernels()
```

Comment:
`isNullKns` or `isNullOrNotNullKns`?

Reply:
renamed `kernels.IsNullKernels()` to `kernels.IsNullNotNullKernels()` to make it more obvious

Reply:
Ah, sorry. I mentioned a local variable name not a function name here.